Feature/book code length

chrisvire opened the pull request:
Fixes #13

mvahowe commented:
Thanks for this. I'd really like to have a maximum length to catch people accidentally passing the entire document as a book code, and I'd prefer to have a minimal length of 3 in case anyone's code relies on there being at least that many characters. Would 3-32 characters work for you?

chrisvire commented:
@mvahowe The file that I added for the test was from a real project that has 2 character book code.  Is that OK?  I updated the PR to have a minimum length of 2 and a maximum length of 32.

mvahowe reviewed (approved):
LGTM - Thanks @chrisvire
